build/webpack.base.config.js

```diff
@@ -19,27 +19,33 @@ console.log(`process.env.NODE_ENV: ${process.env.NODE_ENV}`);
 console.log(`Webpack building for environment: ${environment}`);
 /* eslint-enable no-console */
 
-function getCssLoaders(config) {
+function getCssLoaders(config, loadersAfterCssLoader = 0) {
```

Comment:
`postcss-loader` needs to know how many loaders come after it so I added a param to handle that. Also, each individual return needed `postcss-loader` added to it.